man/man8/zpool.8

```diff
@@ -1449,6 +1458,16 @@ Recovery mode for a non-importable pool. Attempt to return the pool to an import
 .sp
 .ne 2
 .na
+\fB\fB-l\fR\fR
+.ad
+.sp .6
+.RS 4n
+Indicates that the zpool command will request encryption keys for all encrypted datasets it attempts to mount as it is bringing the pool online. This is equivalent to running \fBzfs mount\fR on each encrypted dataset immediately after the pool is imported. If any datasets have a \fBprompt\fR keysource this command will block waiting for the key to be entered. Otherwise, encrypted datasets will be left unavailable until the keys are loaded.
```

Comment:
`zfs key -l` and `zpool import -l` also will result in encrypted zvols appearing in `/dev/`, by the way. To me, these commands allow the user to access the dataset normally doing whatever without worrying about encryption anymore.

That said, I can verify the behavior of this command and make the comment more specific accordingly.